src/processOrders.ts

```
@@ -275,6 +275,14 @@ export const processOrders = async (
                         span.setAttribute("errorDetails", message);
                     }
                     span.setStatus({ code: SpanStatusCode.OK, message });
+                } else if (e.reason === ProcessPairHaltReason.FailedToUpdatePools) {
+                    let message = pair + ": failed to update pool details by event data";
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)